OrbitCore/Utils.h

```diff
@@ -275,6 +275,25 @@ class CWindowsMessageToString {
 };
 #endif
 
+//-----------------------------------------------------------------------------
+inline std::string ShortenStringWithEllipsis(const std::string& text, int max_len) {
+  if (max_len < 0) {
+    return text;
+  }
+  if (max_len <= 3) {
+    return text.length() <= 3 ? text : "...";
+  }
+  if (text.length() <= max_len) {
+    return text;
+  }
+
+  const size_t chars_to_cut =
+      text.length() - static_cast<uint32_t>(max_len) + 3;
```

Comment:
Nit: If I'm not mistaken this cast is not needed here, but I guess the `if` before wants one.

Reply:
I actually changed the overall casts now, casting everything to unsigned after checking for >= 0 in the beginning. The kokoro build was complaining anyways.